keras_cv/layers/preprocessing/random_crop_test.py

```diff
@@ -31,7 +32,7 @@ class RandomCropTest(TestCase):
         ("random_crop_full_width", 3, 8),
     )
     def test_output_shape(self, expected_height, expected_width):
-        np.random.seed(1337)
+        tf.random.set_seed(1337)
```

Comment:
Same here.  Why do we need to use `tf` instead of `np.`?

Reply:
done